includes/miscellaneous.inc

```diff
@@ -99,7 +99,12 @@ function bee_bootstrap($level) {
   }
 
   if (!empty($_bee_backdrop_root) && function_exists('backdrop_bootstrap')) {
+    // Declare Core settings variable so can store settings.
+    global $settings;
+    $settings_initial = $settings;
     backdrop_bootstrap($level);
+    // Restore settings and merge with any newly created settings.
+    $settings = array_unique(array_merge($settings_initial, $settings));
```

Comment:
I would suggest restoring the settings immediately after the config step, so it is present for all remaining bootstrap steps.
```suggestion
    backdrop_bootstrap(BEE_BOOTSTRAP_CONFIG);
    // Restore settings and merge with any newly created settings.
    $settings = array_unique(array_merge($settings_initial, $settings));
    backdrop_bootstrap($level);
```

Backdrop's bootstrap remembers its current level and only runs the remaining ones if `backdrop_bootstrap()` is called multiple times.

Separately, we may have the same problem with the `$config` variable, which would be affected by config overrides specified in settings.php. I believe those are also currently lost.

Reply:
Thanks @quicksketch

I didn't understand this at first, but after digging into the `backdrop_bootstrap` code and the functions that are called at each level this makes some sense as only `_backdrop_bootstrap_configuration`  calls `backdrop_settings_initialize()`.  Another learning experience.

Minor change in my commit as constant should be `BEE_BOOTSTRAP_CONFIGURATION`, not `BEE_BOOTSTRAP_CONFIG`.